integration/test/appcatalog/appcatalogentry/app_catalog_entry_test.go

```diff
@@ -103,7 +103,7 @@ func TestAppCatalogEntry(t *testing.T) {
 			label.AppKubernetesVersion: latestEntry.Version,
 			label.CatalogName:          key.StableCatalogName(),
 			label.CatalogType:          "stable",
-			pkglabel.Latest:            "true",
+			pkglabel.Latest:            "false",
```

Comment:
The test is flakey because the newest prometheus-operator release is a patch.

https://github.com/giantswarm/prometheus-operator-app/releases/tag/v0.7.2

Will fix this as a follow up PR.